### PR TITLE
feat(sm/console): tags and custom props

### PR DESCRIPTION
**Pull request by urbanisierung**
## Description

- https://github.com/camunda/product-hub/issues/2122
- https://github.com/camunda-cloud/camunda-cloud-management-apps/issues/3506

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/self-managed/console-deployment/configuration.md [hunk 1]
```diff
@@ -51,11 +51,13 @@ To enable usage collection, configure the parameters described in the next secti
...
+| ------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- |
+| `customerId`                         | Unique identifier of the customer. This is also a Camunda Docker registry user name                                                                       | `customername`                |
+| `installationId`                     | Unique installation ID of the current customer installation                                                                                               | `my-deployment`               |
+| `telemetry`                          | Telemetry config for Console Self-Managed: `disabled`, `online` or `download`                                                                             | `online`                      |
+| `managed.releases.tags`              | Assign cluster tags to indicate what type of cluster it is. Default tags are `dev`, `stage`, `test` or `prod`, but you can assign any custom tag as well. | `- dev` (list of strings)     |
```
Comment (theburi) [inline docs/self-managed/console-deployment/configuration.md:59]: I would suggest moving this into a different section that discusses connecting the Orchestration Cluster to be managed in the Console.

Comment (urbanisierung) [reply docs/self-managed/console-deployment/configuration.md:59]: @theburi not sure if I understand this correctly. The table describes the config params that can be configured. In console nothing can be really managed so far, it's just a reading view. How about you make a proposal for a new section?

Comment (theburi) [reply docs/self-managed/console-deployment/configuration.md:59]: I find it difficult to understand how to use this property
"managed.releases.tags" or "managed.releases.custom-properties" are not the actual properties I could use.
maybe an example would help here?

File: docs/self-managed/console-deployment/configuration.md [hunk 2]
```diff
@@ -71,6 +73,53 @@ console:
       value: online
 ```
 
+### Override Configuration parameters
```
[the remark this answers is not in the document]
Comment (theburi) [reply docs/self-managed/console-deployment/configuration.md:76]: I think we need a section that gives an example of a config map that Helm creates and then demonstrates how to override configuration and use tags and custom properties.

[the remark this answers is not in the document]
Comment (urbanisierung) [reply docs/self-managed/console-deployment/configuration.md:76]: @theburi added a section with a full example. can you please have another look?

File: docs/self-managed/console-deployment/configuration.md (hunk 1 above)
Comment (urbanisierung) [reply docs/self-managed/console-deployment/configuration.md:59]: @theburi added a section with a full example. can you please have another look?

File: docs/self-managed/console-deployment/configuration.md [hunk 3]
```diff
@@ -71,6 +73,112 @@ console:
...
+                      url: "https://camunda.com"
+                    - name: "Camunda Docs"
+                      url: "https://docs.camunda.io"
+                    - name: "Grafana"
+                      url: "https://..."
+```
+
+### Custom Properties
+
+Custom properties are useful to add custom information to the cluster details page in console. You can mutliple custom properties, and each custom property contains a description and multiple links.
```
Comment (theburi) [inline docs/self-managed/console-deployment/configuration.md:159]: typo, missing 'define'?

**Review by theburi — APPROVED** [review]

File: docs/self-managed/console-deployment/configuration.md (hunk 3 above)
Comment (urbanisierung) [reply docs/self-managed/console-deployment/configuration.md]: Fixed. Thanks for flagging!

**Review by christinaausley — APPROVED** [review]
Nice work @urbanisierung 👍 Just some small tweaks in my latest commit.

**Pull request merged**